Modules/_cursesmodule.c

```diff
@@ -4113,9 +4113,9 @@ NoArgNoReturnFunctionBody(resetty)
 /*[clinic input]
 _curses.resizeterm
 
-    nlines: int
```

Comment:
Can you also update the clinic for `resize_term`? the function is just below.

Reply:
I could have *sworn* that `resize_term` already worked on this patch, I guess not... don't mind me!